test/suite_implementation.js

```diff
@@ -157,6 +157,16 @@ module.exports = function(style, options, _callback) { // eslint-disable-line im
             map.addLayer(new customLayerImplementations[operation[1]](), operation[2]);
             map._render();
             applyOperations(map, operations.slice(1), callback);
+        } else if (operation[0] === 'updateFakeCanvas') {
+            const canvasSource = map.getSource(operation[1]);
+            canvasSource.play();
+            // update before pause should be rendered
+            updateFakeCanvas(window.document, options.addFakeCanvas.id, operation[2]);
```

Comment:
This is putting the logic to be tested in the suite implementation instead of in a specific test. It is not a good precedent, but there is probably not a better way to do this with the current render test framework. Maybe an additional note here tying it back to a specific ticket or render-test would make it easier to remember why it is done this way.

Reply:
Yeah, I agree that expanding the suite implementation further is not great but I'm not seeing a better way. I'm hesitant to add a note pointing to a specific test because notes tend to get outdated. I think searching for "updateFakeCanvas" within tests should be good enough.